src/app/src/test/kotlin/org/imperial/mrc/hint/integration/BaselineTests.kt

```diff
@@ -109,7 +109,7 @@ class BaselineTests : SessionFileTests() {
     @ParameterizedTest
     @EnumSource(IsAuthorized::class)
     fun `can delete shape data`(isAuthorized: IsAuthorized) {
-        val hash = setUpSessionFileAndGetHash(isAuthorized, "malawi.geojson", "/baseline/shape/")
```

Comment:
unrelated! just noticed gradle test runner was warning about unused vars in these tests